lib/old-roman.js

```diff
@@ -0,0 +1,21 @@
+var legend = {1000: 'M',
```

Comment:
love that you're storing these conversions in an object to iterate over them. i wonder if you could be more specific with your variable name though - what does this actually represent?